resources/scripts/bootstrap_menanigfy_vault.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# This script bootstraps the vault
+
+DEFAULT_PASSWORD=changeme
```

Comment:
Default passwords must be excluded from the config file, they can be used as an argument for running the script.